packages/mdc-fab/mdc-fab.scss

```diff
@@ -31,6 +34,24 @@
   @include mdc-fab--mini_;
 }
 
+.mdc-fab--extended {
```

Comment:
Since MDC FAB uses private mixins for most of these selectors' styles, can we do the same for the new ones? It'll result in most of the actual styles being in the same file.